Used templates, created basic layout and components as per design

kevinjung13 opened the pull request:
Used templates from Chakra UI, created basic layout and components as per design

Commit:
Used templates, created basic layout and components as per design

Used templates from Chakra UI, created basic layout and components as per design

jeromevvb commented on package.json:
```diff
@@ -9,18 +9,43 @@
   },
   "devDependencies": {
+<<<<<<< Updated upstream
+=======
```
@kevinjung13 make sure you get rid of your conflicted comments before you commit 
the package.json is not valid for this reason, the build failed on vercel

jeromevvb commented on src/pages/_app.tsx:
```diff
@@ -1,6 +1,49 @@
+<<<<<<< Updated upstream
```
@kevinjung13 same here

jeromevvb commented on src/pages/_app.tsx:
```diff
@@ -1,6 +1,49 @@
+    </ChakraProvider>
+  )
+>>>>>>> Stashed changes
 }
```
@kevinjung13  this also

jeromevvb commented on src/pages/create.tsx:
```diff
@@ -0,0 +1,72 @@
+  
+  const handleClick = () => {
+    <Button
+      isLoading
```
@kevinjung13 
You must use a state, `useState` 
`const [isLoading, setLoading] = useState(false)`
And then when you handle the click, you update the state  `setLoading(true)`

Your button below get the prop isLoading={isLoading}

jeromevvb commented on src/pages/index.tsx:
```diff
@@ -118,6 +115,14 @@ export default function Home() {
+        <link rel='icon' href='/logo.png'/>
+      </Head>
+      <Main />
+>>>>>>> Stashed changes
```
@kevinjung13 same here

jeromevvb commented on src/styles/globals.css:
```diff
@@ -76,6 +77,8 @@
   }
 }
+=======
+>>>>>>> Stashed changes
```
@kevinjung13 thoses comments here

Commit:
Updated files after removing conflicted comments

Removed conflicting comments and added the following:

- create.tsx & mint.tsx: UseState & loading features on button

Commit:
Draft 2

Deleted unnecessary SVG, updated features and organized components and pages into folders

Commit:
Solved Hydration Error, edited links and cleaned up unnecessary code

Solved Hydration Error, edited links and cleaned up unnecessary code.

Code still needs to be cleaned up and more features need to be added

Commit:
Added template for Footer & added new page for successful minting

Added template for Footer & added new page for successful minting

Footer still requires editing, and overall code review will be done

Commit:
Modified code for Header, created draft page for results on minted NFT

Modified code for Header, created draft page for results on minted NFT

Commit:
Fixed Union Expression error

Fixed Union Expression error

Commit:
Created Menu for Connect Wallet and edited code format

Created Menu for Connect Wallet and edited code format

Commit:
Fixed TypeScript Error and changed location of Header interface

Fixed TypeScript Error and changed location of Header interface

Commit:
Changed Logo to be Chakra UI Image and updated links

Changed Logo to be Chakra UI Image to fix rendering error and updated links

Commit:
Updated links for footer, header and buttons due to Vercel error

Updated links for footer, header and buttons due to Vercel error

jeromevvb commented on src/components/Footer/icons.tsx:
```diff
@@ -0,0 +1,70 @@
+  const { children, label, href, _hover } = props;
+
+  return (
+    <chakra.button
```
@kevinjung13 
another approach can be to use the `Link` component from chakra use the props `isExternal` and `href`
This component should wrap the Button

kevinjung13 replied:
@jeromevvb Ah perfect, just tried and it works well so ill do that. Thanks!

jeromevvb commented on src/components/Header/Devices/desktop.tsx:
```diff
@@ -0,0 +1,98 @@
+        <Box key={link.name}>
+          <Popover trigger={'hover'} placement={'bottom-start'} >
+            <PopoverTrigger>
+              <Link
```
If you're using NextJS you should use `next/link` component to navigate through the app
If you a normal `<a>` you force your all app to refresh
try it and see the difference

https://nextjs.org/docs/api-reference/next/link

kevinjung13 replied:
Done!

jeromevvb commented on src/components/Header/index.tsx:
```diff
@@ -0,0 +1,94 @@
+  },
+
+  {
+    name: "Connect Wallet",
```
@kevinjung13 good idea but connect wallet we will replace with https://www.rainbowkit.com/
do you feel capable to integrate it?

jeromevvb commented on src/components/Header/logo.tsx:
```diff
@@ -0,0 +1,20 @@
+      {/* Stacking */}
+      <HStack spacing={8} align={'center'}>
+        {/* Logo */}
+        <Link href={'/main'} _hover={{ opacity: '50%' }}>
```
@kevinjung13 main does not exist should be `/` for index
Same here, make sure to use next/link

kevinjung13 replied:
Done, and I now understand the difference next/link makes. Thanks for sharing!

jeromevvb commented on src/pages/Create/index.tsx:
```diff
@@ -0,0 +1,63 @@
+  
+    /* HandleClick function */
+  const handleClick = () => {
+    setIsLoading(status => !status)
```
@kevinjung13 I need the value containing in the field here ;)

jeromevvb commented on src/pages/Create/index.tsx:
```diff
@@ -0,0 +1,63 @@
+          <Stack spacing={6}>
+            <Button
+              as={'a'}
+              href={'/create/results'}
```
Why are going to a new page? We can handle the request in the same page no?
How would you change your approach?

jeromevvb commented on src/pages/_app.tsx:
```diff
@@ -1,6 +1,59 @@
+/* Themes */
+
+const theme = extendTheme({
+  colors: {
```
@kevinjung13 check the colors system from chakra https://chakra-ui.com/docs/styled-system/theme
we should leverage the colorScheme feature of chakra and provide the full color palette

Commit:
Added isExternal props to Footer icons, changed primary and secondary colors for buttons and added next/links to buttons

- Added isExternal props to Footer icons
- changed primary and secondary colors for buttons
- Added next/links to buttons
- Deleted storybook button component

To do:
- Header links to be fixed
- Color themes to be added
- To go through other comments

Commit:
Created layout component, changed links for buttons and menu options

Created layout component, changed links for buttons and menu options

Remaining:
- Adding results on the "create" page when "create" button is clicked
- Adding RainbowKit for Wallet
- Finishing Suggestion/Example Badge on create form

Commit:
Added clickable badge, onChange for input and  timeout for button

Added clickable badge, onChange for input and  timeout for button

Remaining:
 - On Click, To make results render and hide other components
- To look at implementing RainbowKit

Commit:
Added States for Input, rendering of results for both Create and Mint sections

Added States for Input, rendering of results for both Create and Mint sections
- Remaining: To study Rainbowkit and try to implement

Commit:
Changed Head title

Changed Head title

Commit:
Updated Cards, and Started adding Sections

Updated Cards, and Started adding Sections

Commit:
Added back Connect Wallet on Navbar

Added back Connect Wallet on Navbar

Commit:
changed files names to lowercase

Commit:
Changed CreateSection and Create Page

Added More Badges and Connected input with results

Commit:
Edited MintSection and Minted Sections

Added states to align rendering

Commit:
Modified Mint Section and Mint Page

Added State and Props for URL, Price and Copies

Commit:
Modified Mint, Mint Section and Minted

Fixed issue with Copies rendering. Copies using Steppers to be fixed
